### PR TITLE
Fixes for Python 3 on OS X.

### PR DESCRIPTION
**Pull request by mdickinson**
This PR makes the minimal changes necessary (plus some minor whitespace cleanup) to allow enable to build on Python 3 / OS X.

### CODE REVIEW

**Comment by achabotl** [comment]
What about building the pyside wheel?

**Comment by mdickinson** [comment]
> What about building the pyside wheel?

What about it? Sorry; I'm not sure what you're asking.

**Comment by achabotl** [comment]
Don't you need to build a pyside wheel for Python 3 first? If so the script `build_pyside_wheel.sh` doesn't work because it assumes that the platform is Linux. It worked for me by remove the `--qmake` argument on this line: `python setup.py bdist_wheel --qmake=/usr/bin/qmake-qt4 --version=1.2.2 --jobs=2`. 

Actually, I built the wheel using the [script in the `traitsui` repo](https://github.com/enthought/traitsui/blob/master/build_pyside_wheel.sh), but it applies here to.

**Comment by corranwebster** [comment]
The `build_pyside_wheel.sh` script is for Travis CI support and isn't intended to work on OS X.  The assumption is that you have pyside or pyqt in your environment via some other means for actual deployments.

**Comment by achabotl** [comment]
Got it.

**Comment by mdickinson** [comment]
Marking WIP; I'm tracking down some of the causes of test failures.

**Comment by mdickinson** [comment]
One remaining issue is that the 2to3 fixers are being applied to the SWIG-generated `kiva/agg/agg.py` file. But that file is designed to be Python-agnostic, and running 2to3 on it makes it invalid (it turns references to `__builtin__` into `builtins`). I couldn't find a way to exclude specific files from the 2to3 conversion. A hacky solution would be to exclude the fixer that's making that change, and apply any needed import changes manually in the rest of the source.

**Comment by achabotl** [comment]
Have you looked at `modernize`? It's a wrapper around `2to3`. It tries to use `six` where necessary.

http://python-modernize.readthedocs.io/en/latest/

**Comment by corranwebster** [comment]
To clarify, the problem is really with the setuptools auto-2to3 in the setup.py.

**Comment by mdickinson** [comment]
My `use_2to3_exclude_fixers=['lib2to3.fixes.fix_imports'],` line appears to be being ignored. I haven't figured out why yet.

**Comment by mdickinson** [comment]
Removing the WIP; this is ready for review.

All enable tests and all kiva tests except one pass on my Mac (OS X 10.9), under Python 3.5 from MacPorts. I'm having issues with one kiva test: `test_macport`, but that test is also causing trouble under Python 2, so the issue doesn't appear to be specific to the Python 2 -> Python 3 migration.

**Comment by jwiggins** [comment]
`test_macport` is for WX support, and the compiled extension is there for older versions of WX. I wouldn't worry too much about it.

File: kiva/svg.py [hunk 1]
```diff
@@ -413,7 +413,7 @@ def _emit(self, name, contents=None, kw={}, **otherkw):
 
     def _color(self, color):
         r,g,b,a = color
-        return '#%02x%02x%02x' % (r*255,g*255,b*255)
+        return '#%02x%02x%02x' % (int(r*255),int(g*255),int(b*255))
```
Comment (mdickinson) [inline kiva/svg.py]: This line looks odd to me, but that's orthogonal to this PR. If you want to scale the floating-point range `[0.0, 1.0]` to the integer range `[0, 256)`, doing it in such a way that only the exact value `1.0` maps to `255` and all other values map to something strictly smaller seems wrong.

**Comment by mdickinson** [comment]
@jwiggins 

> I wouldn't worry too much about it.

Thanks! I won't, then. :-)

File: kiva/svg.py (hunk 1 above)
Comment (corranwebster) [inline kiva/svg.py]: Yes, I would agree here from a mathematical point of view.

**Comment by corranwebster** [comment]
LGTM from a reading of the code.  I haven't tried running on OS X, though.

**Comment by mdickinson** [comment]
Merging.

**Pull request merged**